resources/main.mjs

```diff
@@ -331,20 +331,26 @@ class MainBenchmarkClient {
 
     _showSection(hash) {
         if (this._isRunning) {
-            window.location.hash = "#running";
+            this._setLocationHash("#running");
             return;
         } else if (this._hasResults) {
             if (hash !== "#summary" && hash !== "#details") {
-                window.location.hash = "#summary";
+                this._setLocationHash("#summary");
                 return;
             }
         } else {
-            if (hash !== "#home" && hash !== "#about") {
-                window.location.hash = "#home";
+            if (hash !== "" && hash !== "#home" && hash !== "#about") {
+                this._setLocationHash("#home");
                 return;
             }
         }
-        window.location.hash = hash || "#home";
+        this._setLocationHash(hash);
+    }
+
+    _setLocationHash(hash) {
+        if (hash === "#home")
+            hash = "";
```

Comment:
actually, even setting it to `""` will add a `#` to the URL, so it could be good to just not do anything if we want to set it to empty and it's already empty.

Reply:
used now history.pushState to remove the hash in case we're on the home screen.